zebra-network/src/protocol/external/arbitrary.rs

```diff
@@ -112,6 +112,18 @@ impl Message {
     }
 }
 
+impl Arbitrary for Version {
+    type Parameters = ();
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        prop_oneof![170_002_u32..=170_015, 0_u32..]
```

Comment:
Coverage, Low Priority: Can we use the latest current network protocol version here, rather than a hard-coded maximum?

This fix is not important, unless we add version-specific behaviours after `170_015`.

Something like:
```suggestion
        prop_oneof![170_002_u32..=(CURRENT_NETWORK_PROTOCOL_VERSION.0), 0_u32..]
```